### PR TITLE
Fix bird-watcher exercise

### PR DESCRIPTION
**Pull request by porkostomus**
Page still contains embarrassing TODO comment, and it shows the answer to the student. SPOILER ALERT!!

### CODE REVIEW

File: concepts/vectors/introduction.md [hunk 1]
```diff
@@ -1,15 +1,6 @@
...
-TODO: the content below is copied from the exercise introduction and probably needs rewriting to a proper concept introduction
-
-A `vector` in Clojure is a sequential, indexed, immutable collection of zero or more values. This means that once a vector has been created, it cannot be modified. Functions for operating on vectors will return a new vector, while the original vector remains unchanged. The values in a vector may be of heterogenous types. Vectors can be defined as follows:
-
-```clojure
-(def empty [])
-(def single-value 5)
-(def single-value-alternative (vector 5))
-(def three-values [a b c]
-```
```
Comment (ErikSchierboom) [inline concepts/vectors/introduction.md:12]: For concepts that introduce new syntax (like vectors here), we require a syntax example to be included in the introduction. The introduction does not have to be exhaustive though, so maybe include something like:

```clojure
(def empty [])
(def three-values [1 "a" "b"]
```

**Review by ErikSchierboom — APPROVED** [review]
Approved with a small comment.

File: concepts/vectors/introduction.md (hunk 1 above)
Comment (porkostomus) [reply concepts/vectors/introduction.md:12]: That seems sensible :)

**Comment by ErikSchierboom** [comment]
The Clojure track is shaping up great!

**Pull request merged**